src/controllers/questions.js

```diff
@@ -1,13 +1,23 @@
 import Question from '../models/Question';
 
 // Return a list of all questions
-exports.getQuestions = (req, res) => {
-  res.send(Question.queryGetQuestions(req.params.product_id));
+exports.getQuestions = async (req, res) => {
+  // eslint-disable-next-line camelcase
+  const { product_id, limit, page } = req.query;
+  const results = await Question.queryGetQuestions(product_id, limit, page, true);
```

Comment:
Same comment here. Are you hard-coding this argument on purpose?

Reply:
same as above. :)